src/c++/perf_analyzer/README.md

```diff
@@ -73,6 +73,94 @@ Inferences/Second vs. Client p95 Batch Latency
 Concurrency: 1, throughput: 69.6 infer/sec, latency 19673 usec
 ```
 
+## Quick Start
```

Comment:
At some point we need to create a docs folder and split all of this into smaller files. Is that going to happen on a different story? It would be nice to do it sooner rather than later so all future doc links we create are correct.

Reply:
Different story. Let's keep quick start in root readme then more complicated docs in another directory/linked-to from root readme.